scripts/export.js

```diff
@@ -1,7 +1,7 @@
 $(document).ready(() => {
     // set max date for date select to be today
     $.get('/export-users', data => {
-        name_select = $("#name-select")
```

Comment:
it needs to be `let name_select = $("#name-select");`, no? There is no variable declaration or semicolon but it works 👍